app/api/metrics.py

```diff
@@ -1,17 +1,30 @@
-from app.schemas import ServiceMetricCreate
-from fastapi import APIRouter, Depends
+from fastapi.exceptions import RequestValidationError
+from app.schemas import ServiceMetricCreate, MetricCreate
+from fastapi import APIRouter, Depends, Request, exception_handlers, status, Response, HTTPException, FastAPI
+from fastapi.responses import JSONResponse, PlainTextResponse
+from fastapi.encoders import jsonable_encoder
 from pydantic import BaseModel, Field
 from app.crud import CRUDMetric, CRUDServiceMetric, CRUDMicroserviceScoreCard, CRUDMicroservice
-
-from app import dependencies 
+from app import schemas, models, crud, dependencies
+from typing import Any, Callable
+import json
+import re
+from fastapi.routing import APIRoute
+from .exceptions import ExceptionCustom
 from app.core.security import JWTBearer, decodeJWT
 
 class Value(BaseModel):
     value: int | bool | float | str = Field(..., title="Value", description="Value of the metric")
 
 router = APIRouter()
+app = FastAPI()
 
+def format_code(name):
```

Comment:
Extract this function to a file called index.py or something inside a new folder called utils and import it
Make sure to update the microservice.py api file to use the function from the utils file